lib/hybrid-sdk/common/utils/signals.ts

```diff
@@ -1,19 +1,42 @@
-const timers: NodeJS.Timeout[] = [];
+import { getWebsocketConnections } from '../../client';
+import { log as logger } from '../../../logs/logger';
 
+const timers: NodeJS.Timeout[] = [];
+const GRACE_PERIOD_MS = 12000;
```

Comment:
ref: we'll need to update Helm charts if we don't already support graceful shutdown

Reply:
isn't it the standard k8s procedure of giving 30s by default? or you're saying we should have customizable values here?

Reply:
No, I don't think we need to configure this - more just to check the helm charts don't kill the container too early